.eslintrc.js

```diff
@@ -16,6 +16,12 @@ module.exports = {
         '@typescript-eslint/no-shadow': ['error', { builtinGlobals: true }],
       },
     },
+    {
+      files: ['**/*.d.ts'],
+      rules: {
+        'import/unambiguous': 'off',
```

Comment:
Why disable this? :thinking:

Reply:
Wait, on _type definitions_!? How does this rule apply to those :confused:

Reply:
Well, I've looked into this, but I am still confused. No update. Welp.

I'm OK to let this slide for now but I still suspect we're doing something wrong.

Reply:
I _think_ it makes sense given the documentation of `import/unambiguous`: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/unambiguous.md

And indeed, using `parserOptions: { sourceType: 'script' } }` works equally well. Maybe we prefer that?

Reply:
I'm not sure :thinking: I'm curious which part of the docs indicate that this makes sense :sweat_smile: That is, it's not clear to me that a type definition is a module or a script. I don't know which fits better, if either. I was hoping that the TypeScript ESLint module would make some sense from this, but no such luck.

Given my limited understanding at the moment, turning it off seems like the most sensible approach.